apps/desktop/src/components/StackTab.svelte

```diff
@@ -54,6 +104,27 @@
 		&.first {
 			border-radius: var(--radius-ml) 0 0 0;
 		}
+
+		.tab__overflow-menu {
+			opacity: 0;
+			width: 0px;
+			pointer-events: none;
+			transition:
+				opacity 100ms ease-in,
+				width 100ms ease-in-out;
+		}
+
+		.tab__overflow-menu.active,
+		&:active .tab__overflow-menu,
+		&:hover .tab__overflow-menu,
+		&:focus-within .tab__overflow-menu,
+		&:focus .tab__overflow-menu {
+			pointer-events: auto;
+			margin-right: 8px;
+			display: flex;
```

Comment:
This `display: flex` should probably go into `tab__overflow-menu`. It's unclear why it would need to change on active/hover/focus.